sample-key-extractors/src/main/java/com/expedia/www/haystack/pipes/kafkaProducer/extractor/SampleExtractor.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Expedia, Inc.
+ *
+ *       Licensed under the Apache License, Version 2.0 (the "License");
+ *       you may not use this file except in compliance with the License.
+ *       You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *       Unless required by applicable law or agreed to in writing, software
+ *       distributed under the License is distributed on an "AS IS" BASIS,
+ *       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *       See the License for the specific language governing permissions and
+ *       limitations under the License.
+ *
+ */
+package com.expedia.www.haystack.pipes.kafkaProducer.extractor;
+
+import com.expedia.open.tracing.Span;
+import com.expedia.www.haystack.pipes.key.extractor.SpanKeyExtractor;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.util.JsonFormat;
+import com.typesafe.config.Config;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class SampleExtractor implements SpanKeyExtractor {
```

Comment:
Use this for default behaviour of writing whole span as json

Reply:
this change will be done in coming PR